data/base-stealing.yaml

```diff
@@ -5870,49 +5952,55 @@
   town: Shard
   trivial_max: 1750
   trivial_min: 573
-- id: '584'
+- id: 584
   item: white sundress
   item_in: on green counter
   pawnable: false
   room: 13994
   town: Shard
   trivial_max: 1750
   trivial_min: 573
-- id: '585'
+- id: 585
   item: golden sandals
   item_in: on shoe tree
   pawnable: false
   room: 13994
   town: Shard
   vpoorly_max: 573
   vpoorly_min: 573
-- acceptable_max: 305
+- acceptable_max: 336
   acceptable_min: 305
   id: '586'
   item: ruby badge
   item_in: on stand
   pawnable: false
   room: 13998
   town: Shard
+  poorly_max: 576
+  poorly_min: 574
+  vpoorly_max: 609
+  vpoorly_min: 608
 - acceptable_max: 305
   acceptable_min: 305
-  id: '587'
+  id: 587
   item: ceramic thimble
   item_in: in case
   pawnable: false
   room: 13998
   town: Shard
-- acceptable_max: 305
+- acceptable_max: 336
   acceptable_min: 305
-  id: '588'
+  id: 588
   item: target shield
   item_in: on rack
   pawnable: false
   room: 9596
   town: Shard
+  poorly_max: 609
+  poorly_min: 573
 - acceptable_max: 305
   acceptable_min: 305
-  id: '589'
+  id: 589
```

Comment:
Why did this change from string to int? Will that cause problems?

Reply:
The only thing I can think of is that I used ruby 2.3 to run update-stealing. I'll just close this.

Reply:
4cb256ae59f485023a79042f4c4eceda405abae5